ansible_mitogen/loaders.py

```diff
@@ -99,5 +99,5 @@ def assert_supported_release():
 from ansible.plugins.loader import strategy_loader
 
 # These are original, unwrapped implementations
-action_loader__get = action_loader.get
-connection_loader__get = connection_loader.get_with_context
+action_loader__get_with_context = action_loader.get_with_context
+connection_loader__get_with_context = connection_loader.get_with_context
```

Comment:
+1 changing the names. `...__get = ...get_with_context` has long bugged me.

Reply:
Yeah, I saw that and thought like hell I'm leaving it like that hah